### ISSUE TITLE
{@inheritDoc} no longer recognized correctly in FunctionCommentSniff.php

### ISSUE THREAD

**Issue by swiffer** [cakephp/cakephp-codesniffer#293]
```php
/**
 * {@inheritDoc}
 *
 * @OA\Info(title="API Backend", version="0.1")
 */
```

this is giving a warning:

..\Api\AppController.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
----------------------------------------------------------------------
 60 | WARNING | Doc comment for parameter "$event" missing
    |         | (CakePHP.Commenting.FunctionComment.MissingParamTag)

worked fine in 4.0.0

**Comment by ADmad** [comment]
As of v4.1 the `FunctionCommentSniff` has become more strict.

`{@inheritDoc}` is for inheriting docblock comment description only [[1]](https://docs.phpdoc.org/latest/guides/inheritance.html#the-inheritdoc-tag), [[2]](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#42-using-the-inheritdoc-inline-tag-to-augment-a-description).

So you need to add annotations for the method argument(s) and return type too.

**Comment by swiffer** [comment]
Okay, good to know, thanks!

**Comment by othercorey** [comment]
@swiffer You can use just `@inheritDoc` to inherit both comment and tags. Before, all tags were ignored if inheritDoc was anywhere in the comment.
